lib/src/widgets/chat/ds_file_message_bubble.widget.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/controllers/chat/ds_file_message_bubble.controller.dart';
+import 'package:blip_ds/src/widgets/loading/ds_circle_loading.widget.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get_state_manager/get_state_manager.dart';
+
+class DSFileMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final String url;
+  final int size;
+  final String filename;
+  final DSFileMessageBubbleController controller;
+
+  /// Creates a Design System's [DSMessageBubble] used on files other than image, audio, or video
+  DSFileMessageBubble({
+    super.key,
+    required this.align,
+    required this.url,
+    required this.size,
+    required this.filename,
+  }) : controller = DSFileMessageBubbleController();
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () => controller.openFile(filename, url),
+      child: DSMessageBubble(
+        contentPadding: EdgeInsets.zero,
+        align: align,
+        child: SizedBox(
+          height: 80.0,
+          child: Row(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              _buildIcon(),
+              _buildText(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildIcon() {
+    return Container(
+      width: 80.0,
+      color: DSColors.neutralLightSnow,
+      child: Obx(
+        () => controller.isDownloading.value
+            ? const DSCircleLoadingWidget(
+                color: DSColors.neutralDarkRooftop,
+              )
+            : Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Image.asset(
+                    controller.getAsset(filename),
+                    height: 42.0,
+                    package: DSUtils.packageName,
+                    errorBuilder: (_, __, ___) => Image.asset(
+                      'assets/images/file-default.png',
+                      package: DSUtils.packageName,
+                    ),
+                  )
+                ],
+              ),
+      ),
+    );
+  }
+
+  Widget _buildText() {
+    final color = align == DSAlign.right
+        ? DSColors.neutralLightSnow
+        : DSColors.neutralDarkCity;
+
+    return Flexible(
+      child: Container(
+        padding: const EdgeInsets.symmetric(
+          vertical: 8.0,
+          horizontal: 16.0,
+        ),
+        constraints: const BoxConstraints(
+          minWidth: 160.0,
+        ),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            DSBodyText(
```

Comment:
We need to set textAlign to center to avoid this:
<img width="314" alt="Screen Shot 2022-08-04 at 11 28 53" src="https://user-images.githubusercontent.com/20003773/182872369-23ddc2f0-f791-4184-8d22-4a9309154db1.png">

This prop must be added to the DSBodyText
![Screen Shot 2022-08-04 at 11 30 07](https://user-images.githubusercontent.com/20003773/182873450-742085e8-e851-448d-9aea-7ca2f6be704f.png)
![Screen Shot 2022-08-04 at 11 30 17](https://user-images.githubusercontent.com/20003773/182873474-316da13e-ad83-489e-99c2-640a8ec3796a.png)
![Screen Shot 2022-08-04 at 11 30 29](https://user-images.githubusercontent.com/20003773/182873485-9d99745c-4002-40e3-87cd-a74132f711b5.png)

Reply:
done!